### ISSUE TITLE
Directive @extends status

### ISSUE THREAD

**Issue by Robert-Scub** [gql-dart/ferry#322]
Hello!

Thank you for this superb API.

My backend is generating a schema that contains this directive declaration:

`directive @extends on OBJECT | INTERFACE`

I need to remove it or else it break Flutter code generation.

`line 1, column 266 of .: 'extends' can't be used as an identifier because it's a keyword.`

Thanks!

**Comment by mit-73** [comment]
I came across this too, most likely it should be discussed in the [gql](https://github.com/gql-dart/gql) repository

**Comment by knaeckeKami** [comment]
Yes, please open an issue in the gql repo, as this is the place where the parsing of the schema is implemented.
